crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -619,6 +669,30 @@ module deepbook::clob_v2 {
         (coin::from_balance(base_asset_balance, ctx), coin::from_balance(quote_asset_balance, ctx), val)
     }
 
+    public fun swap_exact_quote_for_base_with_metadata<BaseAsset, QuoteAsset>(
+        pool: &mut Pool<BaseAsset, QuoteAsset>,
+        client_order_id: u64,
+        account_cap: &AccountCap,
```

Comment:
same question for the 2 args being swapped. Not a big deal again, wondering...

Reply:
swap_exact_quote_for_base_with_metadata is a function based on original function swap_exact_quote_for_base. Hence, param ordering is followed as per swap_exact_quote_for_base function instead of place_market_order. Lemme know if you see any inconsistencies with this.